src/libcollections/btree/node.rs

```diff
@@ -1140,66 +1042,56 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Internal>, marker::
             let height = self.node.height;
             let new_len = self.node.len() - self.idx - 1;
 
-            ptr::copy_nonoverlapping(
-                self.node.keys().as_ptr().offset(self.idx as isize + 1),
-                new_node.data.keys.as_mut_ptr(),
-                new_len
-            );
-            ptr::copy_nonoverlapping(
-                self.node.vals().as_ptr().offset(self.idx as isize + 1),
-                new_node.data.vals.as_mut_ptr(),
-                new_len
-            );
-            ptr::copy_nonoverlapping(
-                self.node.as_internal().edges.as_ptr().offset(self.idx as isize + 1),
-                new_node.edges.as_mut_ptr(),
-                new_len + 1
-            );
+            ptr::copy_nonoverlapping(self.node.keys().as_ptr().offset(self.idx as isize + 1),
+                                     new_node.data.keys.as_mut_ptr(),
+                                     new_len);
+            ptr::copy_nonoverlapping(self.node.vals().as_ptr().offset(self.idx as isize + 1),
+                                     new_node.data.vals.as_mut_ptr(),
+                                     new_len);
+            ptr::copy_nonoverlapping(self.node
+                                         .as_internal()
+                                         .edges
+                                         .as_ptr()
+                                         .offset(self.idx as isize + 1),
+                                     new_node.edges.as_mut_ptr(),
+                                     new_len + 1);
 
             self.node.as_leaf_mut().len = self.idx as u16;
             new_node.data.len = new_len as u16;
 
             let mut new_root = Root {
                 node: BoxedNode::from_internal(new_node),
-                height: height
+                height: height,
             };
 
-            for i in 0..(new_len+1) {
+            for i in 0..(new_len + 1) {
                 Handle::new_edge(new_root.as_mut().cast_unchecked(), i).correct_parent_link();
             }
 
-            (
-                self.node,
-                k, v,
-                new_root
-            )
+            (self.node, k, v, new_root)
         }
     }
 
     /// Returns whether it is valid to call `.merge()`, i.e., whether there is enough room in
     /// a node to hold the combination of the nodes to the left and right of this handle along
     /// with the key/value pair at this handle.
     pub fn can_merge(&self) -> bool {
-        (
-            self.reborrow()
-                .left_edge()
-                .descend()
-                .len()
-          + self.reborrow()
-                .right_edge()
-                .descend()
-                .len()
-          + 1
-        ) <= CAPACITY
+        (self.reborrow()
+             .left_edge()
+             .descend()
+             .len() +
```

Comment:
I guess this is just personal style, but this seems to make the `+` hard to notice - I initially thought that this was a tuple based on this style and got confused.